### DISCUSSION TITLE
Welcome to openai-whisper Discussions!

### DISCUSSION THREAD

**Discussion by nyadla-sys** [usefulsensors/openai-whisper#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
hello, is there a slack group to join? I'd like to try out this whisper thing on android phone and googled a bit and landing here.

**Reply by nyadla-sys** [reply]
Please refer the below code to have its android implementation

https://github.com/usefulsensors/openai-whisper/tree/main/android_app

**Comment by yenerismail** [comment]
Hello,
I downloaded your app to be used on my Android phone. But there is only English language.
I live in Türkiye and speak Turkish. I would appreciate if you could change your Whisper model.
Enjoy your work,
Android10 and Android13 use.
